sematic/db/models/run.py

```diff
@@ -97,7 +97,7 @@ class Run(HasUserMixin, Base, JSONEncodableMixin):
         types.String(), nullable=False, info={ENUM_KEY: FutureState}
     )
     name: str = Column(types.String(), nullable=True)
-    calculator_path: str = Column(types.String(), nullable=False)
+    function_path: str = Column(types.String(), nullable=False)
```

Comment:
This rename will make it so that old API clients against a new API will get a payload that has a `function_path` field, when they are expecting a `calculator_path` to be able to properly deserialize. We can probably modify `get_run_payload` in `payloads.py` to copy the `"function_path"` field to the `"calculator_path"` field in the response to remedy this mismatch and provide backwards compatibility.